ci/docker-pull-or-build-and-push.sh

```diff
@@ -7,6 +7,11 @@ DIRECTORY_HASH=$(git rev-parse HEAD:ci/$image_name)
 export DIRECTORY_HASH
 cd ci/$image_name/
 
+export USER_ID=3000
+export GROUP_ID=3000
+sudo groupadd -g $USER_ID docker_user
```

Comment:
```suggestion
sudo groupadd -g $GROUP_ID docker_user
```
Both are equal, but in general I think that was your intention!